modules/RTC/BridgeChannel.js

```diff
@@ -170,6 +170,20 @@ export default class BridgeChannel {
     }
 
     /**
+     * Sends a "receiver video constraint" message via the channel.
+     * @param {Number} maxFrameHeightPixels the maximum fram height,
+     * in pixels, this receiver is willing to receive
+     */
+    sendReceiverVideoConstraintMessage(maxFrameHeightPixels) {
```

Comment:
Kind of strong, unless you feel even stronger about not removing it :) We don't prepend lastN, nor selected endpoint, for instance, and the idea there is exactly the same.